src/dispatching/dispatchers/mod.rs

```diff
@@ -1,3 +1,5 @@
+//! Different types of dispatchers.
```

Comment:
But we have only one, and it's not clear that we will have more in the future...

Reply:
We'll have the FSM-based dispatcher in future, and probably an actor-based.

Reply:
I think that `FSM`-based is just filter with `FSMFilter`

Reply:
I'll think on it.